modules/op-running-unpriveleged-builds-in-a-container.adoc

```diff
@@ -0,0 +1,51 @@
+[id='op-running-unpriveleged-builds-in-a-container._{context}']
+= Running unprivileged builds in a container
+
+You can run builds as a non-root user inside a container. To run as a non-root user inside the container, your container must first add the `subuid` and `subgid` with the latest shadow-utils.
+
+The public image based on Red Hat UBI has everything already set up for us in its https://catalog.redhat.com/software/containers/ubi8/buildah/602686f7b16b1eb2e30807ee?container-tabs=dockerfile[Containerfile]. As an extra step, you need to “force” running the container as the “build” user since if we leave it to the default, it will be “randomed” and would not have the setup needed to run as non-root.
```

Comment:
Typically, you only want to embed links in additional resources or preqs, but even be careful there. Clicking links takes users away from your documentation. 

Also, avoid `us` and `we`. Like the IBM SG wants, focus exclusively on the user.